lib/truncate.js

```diff
@@ -362,7 +389,15 @@ function truncateCustomKeys (value, max, keywords) {
   }
   const result = value
   const keys = Object.keys(result)
-  const truncatedKeys = keys.map(k => keywords.includes(k) ? k : truncate(k, max))
+  const truncatedKeys = keys.map(k => {
```

Comment:
👍 for breaking up the one liner and adding the `if (k.length > max) {` check before truncating.